doc/reference/storage_ceph.md

```diff
@@ -1,56 +1,110 @@
 (storage-ceph)=
-# Ceph - `ceph`
-
-- Uses RBD images for images, then snapshots and clones to create instances
-  and snapshots.
-- Due to the way copy-on-write works in RBD, parent filesystems can't be
-  removed until all children are gone. As a result, LXD will automatically
-  prefix any removed but still referenced object with "zombie_" and keep it
-  until such time the references are gone and it can safely be removed.
-- Note that LXD will assume it has full control over the osd storage pool.
-  It is recommended to not maintain any non-LXD owned filesystem entities in
-  a LXD OSD storage pool since LXD might delete them.
-- Note that sharing the same osd storage pool between multiple LXD instances is
-  not supported. LXD only allows sharing of an OSD storage pool between
-  multiple LXD instances only for backup purposes of existing instances via
-  `lxd import`. In line with this, LXD requires the "ceph.osd.force_reuse"
-  property to be set to true. If not set, LXD will refuse to reuse an osd
-  storage pool it detected as being in use by another LXD instance.
-- When setting up a Ceph cluster that LXD is going to use we recommend using
-  `xfs` as the underlying filesystem for the storage entities that are used to
-  hold OSD storage pools. Using `ext4` as the underlying filesystem for the
-  storage entities is not recommended by Ceph upstream. You may see unexpected
-  and erratic failures which are unrelated to LXD itself.
-- To use Ceph osd pool of type "erasure" you __must__ have the osd pool created
-  beforehand, as well as a separate osd pool of type "replicated" that will be used for
-  storing metadata. This is required as RBD & CephFS do not support omap.
-  To specify which pool is "earasure coded" you need to use the
-  `ceph.osd.data_pool_name=<erasure-coded-pool-name>` and
-  `source=<replicated-pool-name>` for the replicated pool.
-
-## Storage pool configuration
+# Ceph RBD - `ceph`
+
+<!-- Include start Ceph intro -->
+
+```{youtube} https://youtube.com/watch?v=kVLGbvRU98A
+```
+
+[Ceph](https://ceph.io/en/) is an open-source storage platform that stores its data in a storage cluster based on {abbr}`RADOS (Reliable Autonomic Distributed Object Store)`.
```

Comment:
Maybe just https://ceph.io/ here in case they change their URL formats in the future?